backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product is intended to safeguard customer applications on the OpenShift Platform. OADP provides disaster recovery protection for OpenShift applications, application related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
+
+OADP is not intended to be a disaster recovery solution for *etcd* or OpenShift Operators. For more information on etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
+
+There exists a multitude of approaches to customize a backup to avoid backing up inappropriate resources by using namespaces or labels.
```

Comment:
I am not aware of the technicalities. But are those approaches = the following list of APIs?
If so, my suggestion:
```suggestion
OAP provides the following APIs to enable multiple approaches to customize a backup to avoid backing up inappropriate resources by using namespaces or labels.
```
else
 you might have to explain those multiple approaches.

Reply:
(giggle) @snarayan-redhat - you spelt OADP wrong! What a bad reviewer! LOL

Seriously, thanks for the great help you gave with your review.